.gitignore

```diff
@@ -24,7 +24,12 @@ lib64
 CL.txt
 /venv
 #/doc/source/user_guide*
-#!/doc/source/user_guide/*rst
+/doc/source/user_guide/*.png
+/doc/source/user_guide/*.fits
+/doc/source/user_guide/*.root
+/doc/source/user_guide/ScienceWindowList-Copy1.ipynb
```

Comment:
why these lines?

Reply:
files generated with the execution of the example notebooks

Reply:
`ScienceWindowList-Copy1.ipynb` is generated? could in principle be, but where?

Reply:
this file was simply a test file, and ended up here as a mistake (most likely automatically added by the IDE)